playbooks/roles/ocp-metallb-operator/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# vars file for playbooks/roles/ocp-metallb-operator
+metallb_e2e_env:
+  PATH: "/usr/local/go/bin:{{ ansible_env.PATH }}"
+  KUBECONFIG: "{{ ansible_env.HOME }}/.kube/config"
+  IS_OPENSHIFT: true
+  NAMESPACE: metallb-system
```

Comment:
Why are we not using this env everywhere? I see them harcoded.

Reply:
This env file is passed only to the e2e tests from metallb-operator repo.